README.md

```diff
@@ -53,27 +53,38 @@ The following example reads in data from the heat pump:
 from luxtronik import Luxtronik
 
 l = Luxtronik('192.168.1.23', 8889)
-calculations, parameters, visibilities = l.read()
+heating_limit = l.parameters.get("ID_Einst_Heizgrenze")
```

Comment:
The ID of the temperature of the heating limit is `ID_Einst_Heizgrenze_Temp`. This is probably meant here. I am not sure, what `ID_Einst_Heizgrenze` is. Maybe a `bool` whether the "Heizgrenze" is enabled?